consensus/poet/core/tests/test_consensus/test_poet_fork_resolver.py

```diff
@@ -0,0 +1,649 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+import shutil
+import tempfile
+
+from importlib import reload
+
+from unittest import TestCase
+from unittest import mock
+
+from sawtooth_poet.poet_consensus import poet_fork_resolver
+
+from sawtooth_poet_common.protobuf.validator_registry_pb2 \
+    import ValidatorInfo
+from sawtooth_poet_common.protobuf.validator_registry_pb2 \
+    import SignUpInfo
+
+
+@mock.patch('sawtooth_poet.poet_consensus.poet_block_publisher.BlockWrapper')
+@mock.patch('sawtooth_poet.poet_consensus.poet_block_publisher.PoetConfigView')
+class TestPoetBlockResolver(TestCase):
```

Comment:
Recommend renaming class to `TestPoetForkResolver` to match both file name and module it is testing.